content/blog/introducing-pulumi-a-cloud-development-platform/index.md

```diff
@@ -1,18 +1,16 @@
 ---
-title: "TODO Port frontmatter"
-authors: ["chris-smith"]
-tags: ["todo"]
-date: "2017-01-01"
-draft: true
-description: "TODO: Put in a reasonable summary"
----
+title: "Introducing Pulumi, a Cloud Development Platform"
+authors: ["joe-duffy"]
+tags: ["Pulumi"]
+date: "2018-06-18"
```

Comment:
This should be the first post, but some other ones that were also published on `2018-06-18` are showing up before it. Is there a way to ensure this one is first, and the other ones are in the right order, perhaps by adding times to the posts published on `2018-06-18`?

Reply:
I just changed the date of the those other posts to be +1-3 days.